<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>resume</title>
  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <style type="text/css">
  /*
   * Copyright 2013 Christophe-Marie Duquesne <chmd@chmd.fr>
   *
   * CSS for making a resume with pandoc. Inspired by moderncv.
   *
   * This CSS document is delivered to you under the CC BY-SA 3.0 License.
   * https://creativecommons.org/licenses/by-sa/3.0/deed.en_US
   */
  
  /* Whole document */
  body {
      font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
      max-width: 800px;
      margin: auto;
      background: #FFFFFF;
      padding: 10px 10px 10px 10px;
  }
  
  /* Title of the resume */
  h1 {
      font-size: 55px;
      color: #757575;
      text-align:center;
      margin-bottom:15px;
  }
  h1:hover {
      background-color: #757575;
      color: #FFFFFF;
      text-shadow: 1px 1px 1px #333;
  }
  
  /* Titles of categories */
  h2 {
      /* This is called "sectioncolor" in the ConTeXt stylesheet. */
      color: #397249;
  }
  /* There is a bar just before each category */
  h2:before {
      content: "";
      display: inline-block;
      margin-right:1%;
      width: 16%;
      height: 10px;
      /* This is called "rulecolor" in the ConTeXt stylesheet. */
      background-color: #9CB770;
  }
  h2:hover {
      background-color: #397249;
      color: #FFFFFF;
      text-shadow: 1px 1px 1px #333;
  }
  
  /* Definitions */
  dt {
      float: left;
      clear: left;
      width: 17%;
      /*font-weight: bold;*/
  }
  dd {
      margin-left: 17%;
  }
  p {
      margin-top:0;
      margin-bottom:7px;
  }
  
  /* Blockquotes */
  blockquote {
      text-align: center
  }
  
  /* Links */
  a {
      text-decoration: none;
      color: #397249;
  }
  a:hover, a:active {
      background-color: #397249;
      color: #FFFFFF;
      text-decoration: none;
      text-shadow: 1px 1px 1px #333;
  }
  
  /* Horizontal separators */
  hr {
      color: #A6A6A6;
  }
  
  table {
      width: 100%;
  }
  </style>
</head>
<body>
<p><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous"></p>
<h1 id="zijun-zhang">Zijun Zhang</h1>
<table>
<tbody>
<tr class="odd">
<td>Bioinformatics IDP, UCLA</td>
<td style="text-align: right;"><a href="mailto:zj.z@ucla.edu">zj.z@ucla.edu</a></td>
</tr>
<tr class="even">
<td>Los Angeles, CA90095</td>
<td style="text-align: right;"><a href="https://zj-zhang.github.io/" class="uri" target="_blank">https://zj-zhang.github.io/</a></td>
</tr>
<tr class="odd">
<td>USA</td>
<td style="text-align: right;"></td>
</tr>
</tbody>
</table>
<h2 id="education"><i class="fa fa-graduation-cap"></i> Education</h2>
<dl>
<dt>2014-2019 (expected)</dt>
<dd>Ph.D.Â in Bioinformatics; UCLA (Los Angeles, USA)
</dd>
<dt>2010-2014</dt>
<dd>B.Sc. in Bioinformatics; Zhejiang University (Hangzhou, China)
</dd>
</dl>
<h2 id="publications"><i class="fas fa-file-alt"></i> Publications</h2>
<ul>
<li><strong>Zhang, Zijun</strong>; Park, Eddie; Lin, Lan; Xing, Yi; <a>A panoramic view of RNA modifications: exploring new frontiers</a> (Genome Biology, 2018)</li>
<li>Park, Eddie; Pan, Zhicheng; <strong>Zhang, Zijun</strong>; Lin, Lan; Xing, Yi; <a>The expanding landscape of alternative splicing variation in human populations</a> (American Journal of Human Genetics, 2018)</li>
<li><strong>Zhang, Zijun</strong>; Xing, Yi; <a>CLIP-seq analysis of multi-mapped reads discovers novel functional RNA regulatory sites in the human transcriptome</a> (Nucleic acids research, 2017)</li>
<li>Liu, Lili; <strong>Zhang, Zijun</strong>; Sheng, Taotao; Chen, Ming; <a>DEF: an automated dead-end filling approach based on quasi-endosymbiosis</a> (Bioinformatics, 2017)</li>
<li>Liu, Lili; Mei, Qian; Yu, Zhenning; Sun, Tianhao; <strong>Zhang, Zijun</strong>; Chen, Ming; <a>An integrative bioinformatics framework for genome-scale multiple level network reconstruction of rice</a> (Journal of integrative bioinformatics,2013)</li>
<li>Liu, Lili; <strong>Zhang, Zijun</strong>; Mei, Qian; Chen, Ming; <a>PSI: a comprehensive and integrative approach for accurate plant subcellular localization prediction</a> (PLoS One, 2013)</li>
</ul>
<h2 id="projects"><i class="fab fa-github"></i> Projects</h2>
<dl>
<dt><strong>CLAM</strong></dt>
<dd><em>CLIP-seq Analysis of Multi-mapped reads</em>
</dd>
<dd>Open-source python package for calling peaks in CLIP/RIP-seq data.
</dd>
<dt><strong>DARTS</strong></dt>
<dd><em>Deep learning augmented Analysis of RNA-seq Transcript Splicing</em>
</dd>
<dd>Bayesian inference utilizing deep-learning prediction informative prior
</dd>
</dl>
<h2 id="skills"><i class="fas fa-cogs"></i> Skills</h2>
<ul>
<li>Programming languages: Python, R, scikit, Keras, Tensorflow, Theano, Matlab, AWS, HTML, CSS, PHP, Jekyll, SQL, C/C++, Unix/SGE, Git</li>
<li>Statistics: Regularized regression, Bayesian statistics and Bayesian inference, Causal Inference, Hierarchical modeling, MCMC, multivariate analysis</li>
<li>Machine/Deep learning: Supervised(CNN, LSTM, adaboost/xgboost, SVM, random forest), Unsupervised(clustering, autoencoder, factor analysis), Reinforcement(controller network)</li>
</ul>
</body>
</html>
